src/middlewared/middlewared/alembic/versions/12.0/2020-05-18_14-29_nmbclusters-fix.py

```diff
@@ -84,7 +85,7 @@ def upgrade():
     #   1. this is a freeBSD system
     #   2. kern.ipc.nmbclusters is a "sysctl" type entry in the db
     #        AND is < the current OS value
-    if osc.IS_FREEBSD:
+    if platform.system().upper() == 'FREEBSD':
```

Comment:
This will never be true, these migrations are always executed on Linux

Reply:
@themylogin should we remove the complete logic of the migration then?